csrc/fusion.cpp

```diff
@@ -26,21 +26,25 @@
 
 namespace nvfuser {
 
-static thread_local Fusion* ACTIVE_FUSION = nullptr; // NOLINT
+/*static*/ thread_local const Fusion* FusionGuard::active_fusion_ = nullptr;
```

Comment:
@wujingyue to be clear: I am not sure why we need to introduce `const` in this line.

Reply:
I missed that. Fixed now.